web/src/modules/layout/containers/LocationLayout.js

```diff
@@ -102,17 +101,14 @@ export class LocationLayout extends React.Component<PropsType, LocalStateType> {
   }
 
   render () {
-    const { viewportSmall, children, location, darkMode, languageChangePaths, events } = this.props
+    const { viewportSmall, children, location, darkMode, languageChangePaths, events, isLoading } = this.props
     const type = location.type
     const { city, language } = location.payload
 
     const cityModel = this.getCurrentCity()
 
     if (!cityModel) {
-      return <Layout header={<GeneralHeader viewportSmall={viewportSmall} />}
-                     footer={<GeneralFooter />}>
-        {children}
-      </Layout>
+      return null
```

Comment:
Not sure whether this is the best solution. Other options:
* Show loading children (LoadingSpinner) as long as the city model is not available -> LoadingSpinner is pushed down once there is a city model
* Show only LoadingSpinner as long as it is loading -> long loading time until something happens

Reply:
I also like the first option. But maybe it would also seem a little bit too "jumpy" so i am also not quite sure...

Reply:
Have you tried out implementing one of the other options?

Reply:
Yes, tried both. I think any option has its advantages. With the first option I don't like that the goal was to prevent jumping behavior and now we are introducing jumping behavior of the loading spinner. Not sure if that is a problem though. @Schedulaar what do you think?

Reply:
I don't really get the issue about the cityModel. It is posisble to render the header without the cityModel. If the cityModel is not yet available then this header is shown:

![image](https://user-images.githubusercontent.com/905221/95578866-eadf1e00-0a34-11eb-8305-b79c5f8866ea.png)

Reply:
I cannot really tell what is the best solution. You might try analyzing it with Lighthouse on several pages (e.g. /landkreisgarmischpartenkirchen/de/corona-virus/informationen-zum-corona-virus) on your own. Doing so with the desktop config gives me a 0.254 CLS value. You might want if that improved(=reduced) with your method.

Reply:
@maxammann doesn't really look great imo. Also, if only categories are enabled, the header will collapse later.

Reply:
Tested CLS for mentioned methods. The implemented one (show nothing as long as city model is not available) ranks best with a CLS of 0.039. Option 1 (show loading spinner as long as no city model is available) has a CLS of 0.069. Before the changes CLS was 1. So I guess we should not change anything further.